2022-01-26 10:21:23,586 INFO    MainThread:6391 [wandb_setup.py:_flush():71] setting env: {}
2022-01-26 10:21:23,586 INFO    MainThread:6391 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-26 10:21:23,586 INFO    MainThread:6391 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220126_102123-14ttzgu0/logs/debug.log
2022-01-26 10:21:23,586 INFO    MainThread:6391 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220126_102123-14ttzgu0/logs/debug-internal.log
2022-01-26 10:21:23,587 INFO    MainThread:6391 [wandb_init.py:init():404] calling init triggers
2022-01-26 10:21:23,587 INFO    MainThread:6391 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-26 10:21:23,587 INFO    MainThread:6391 [wandb_init.py:init():460] starting backend
2022-01-26 10:21:23,587 INFO    MainThread:6391 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-26 10:21:23,604 INFO    MainThread:6391 [backend.py:ensure_launched():216] starting backend process...
2022-01-26 10:21:23,619 INFO    MainThread:6391 [backend.py:ensure_launched():222] started backend process with pid: 6446
2022-01-26 10:21:23,620 INFO    MainThread:6391 [wandb_init.py:init():469] backend started and connected
2022-01-26 10:21:23,622 INFO    MainThread:6391 [wandb_init.py:init():533] updated telemetry
2022-01-26 10:21:23,662 INFO    MainThread:6391 [wandb_init.py:init():563] communicating current version
2022-01-26 10:21:24,210 INFO    MainThread:6391 [wandb_init.py:init():568] got version response 
2022-01-26 10:21:24,211 INFO    MainThread:6391 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-26 10:21:24,450 INFO    MainThread:6391 [wandb_init.py:init():606] starting run threads in backend
2022-01-26 10:21:25,917 INFO    MainThread:6391 [wandb_run.py:_console_start():1810] atexit reg
2022-01-26 10:21:25,918 INFO    MainThread:6391 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-26 10:21:25,918 INFO    MainThread:6391 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-26 10:21:25,919 INFO    MainThread:6391 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-26 10:21:25,919 INFO    MainThread:6391 [wandb_init.py:init():633] run started, returning control to user process
2022-01-26 10:21:26,107 INFO    MainThread:6391 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-26 10:21:26,107 INFO    MainThread:6391 [wandb_run.py:_restore():1752] restore
2022-01-26 10:21:28,436 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-01-26 10:21:28,781 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-01-26 10:21:29,465 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-01-26 10:21:29,568 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12485
}

2022-01-26 10:21:29,670 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12485
}

2022-01-26 10:21:29,773 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12485
}

2022-01-26 10:21:29,877 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:29,981 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:30,086 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:30,191 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:30,297 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:30,859 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}

2022-01-26 10:21:31,382 INFO    MainThread:6391 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12485
  total_bytes: 12485
}
local_info {
}

2022-01-26 10:21:32,452 INFO    MainThread:6391 [wandb_run.py:_append_files():2180] logging synced files
